options/db_options.cc

```diff
@@ -20,6 +20,29 @@
 
 namespace ROCKSDB_NAMESPACE {
 #ifndef ROCKSDB_LITE
+static std::unordered_map<std::string, WALRecoveryMode>
```

Comment:
If these enum maps are used in other .cc files, user may need to add the declaration. Do you think if you add the declaration of these enums to the corresponding .h files will be easy to use (similar as in options_help.h)?

Reply:
I do not believe the enum maps will be required outside of this file.  For the most part, I do not think I changed the scope of where any of these enum maps were required/used.

Reply:
> I do not believe the enum maps will be required outside of this file. For the most part, I do not think I changed the scope of where any of these enum maps were required/used.

Got it. Thanks!